app/hooks/use-display-currency.ts

```diff
@@ -81,12 +90,23 @@ export const useDisplayCurrency = () => {
     [fiatSymbol],
   )
 
+  // TODO: remove
+  const computeUsdAmount = (tx: Transaction) => {
```

Comment:
This does not seem to belong to in `useDisplayCurrency`. It will also not always be the UsdAmount. It is actually computing the amount in display currency.

Reply:
same comment as here: https://github.com/GaloyMoney/galoy-mobile/pull/1275#discussion_r1119101823